cmd/common/util_test.go

```diff
@@ -1,29 +1,33 @@
 package common
 
 import (
+	"github.com/cosmos/cosmos-sdk/client"
```

Comment:
sort imports?

Reply:
I think I did, I'll run `gofmt` again to make sure things are in the order that is used by it.